.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -28,9 +28,30 @@ jobs:
 
           echo "Latest commit on 'staging' branch is: $(git log -1 --format='%H' staging)"
 
-          echo "Latest commit on 'main' branch is: $(git log -1 --format='%H' main)"
+          # echo "Latest commit on 'main' branch is: $(git log -1 --format='%H' main)"
+          echo "Latest commit on 'psridharan/trigger-stage-deploy' branch is: $(git log -1 --format='%H' psridharan/trigger-stage-deploy)"
 
-          echo "About to merge 'main' branch into 'staging' branch"
-          git merge --verbose main -m "Merging main branch into staging branch"
+          # echo "About to merge 'main' branch into 'staging' branch"
+          echo "About to merge 'psridharan/trigger-stage-deploy' branch into 'staging' branch"
+          # git merge --verbose main -m "Merging main branch into staging branch"
+          git merge --verbose psridharan/trigger-stage-deploy -m "Merging psridharan/trigger-stage-deploy branch into staging branch"
 
           git push origin staging
+      - name: Trigger repository_dispatch to build images
+        if: success()
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          event-type: build-images
+      #- name: Trigger repository_dispatch to deploy images
+      #  if: success()
+      #  uses: peter-evans/repository-dispatch@v2
+      #  with:
+      #    token: ${{ secrets.GITHUB_TOKEN }}
+      #    event-type: deploy-images
+      #- name: Trigger repository_dispatch to run unit tests
```

Comment:
Although I don't think we need to run unit tests on staging (per earlier comment), if we _do_ run them, we would want to do that before building and deploying images, right?

Reply:
@atolopko-czi - Will remove it entirely and we can revisit later whether we want to run tests or not based on what we observe